<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="11.01s"
		durationms="11009"
		type="root"
		rid="35229239270ae4a"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="voillat"
		rxversion="12.1.1.0"
		host="PL1CHZRH0298NB"
		osversion="Windows 11 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-CH"
		screenresolution="3840x1080"
		timestamp="14/02/2025 14:25:56"
		timestampiso="2025-02-14T13:25:56"
		endtime="14/02/2025 14:26:07"
		timeoutfactor="1"
		runid=""
		sutversion="">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:11.005"
			timeRelativeToTestModuleStartTime="00:11.005"
			timeWallClock="14:26:07"
			time="00:11.005"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
